.github/workflows/wdio-tests.yml

```
@@ -60,5 +60,37 @@ jobs:
           PADDLE_API_KEY: ${{ secrets.PADDLE_API_KEY }}
           QUOTA_COUNT_KV_ATTRIBUTES: true
           QUOTA_COUNT_LT_ATTRIBUTES: true
+        run: |
+          npm run wdio:fast
+
+  wdio-tests-pglite:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)